src/pkg/cli/common.go

```diff
@@ -38,6 +38,6 @@ func PrintObject(root string, data proto.Message) error {
 		return err
 	}
 	// TODO: add color
-	fmt.Println(string(bytes))
+	term.Info(string(bytes))
```

Comment:
This will make the whole thing purple. revert. 

The comment "add color" was meant to add some sort of highlighting when printing the yaml.

Reply:
I actually didnt' do it to make it purple. I did it so I could make assertions on the output buffer. How do you suggest I print with term without adding colour?